client/assets/stylesheets/shared/_typography.scss

```diff
@@ -21,6 +21,14 @@ $signup-sans: 'Noto Sans', $sans;
 	color: var( --color-text );
 }
 
+@font-face {
+	font-display: swap;
+	font-family: 'Recoleta';
```

Comment:
Would it be safer to provide the font-fallback here? I'm not sure if all folks realize that fonts may not support all languages.

Reply:
[Seems we cannot set fallbacks inside the `@font-face` definition.](https://stackoverflow.com/questions/26927564/adding-fallback-fonts-to-the-font-face-definition).